apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -1,3 +1,8 @@
+export const UploadType = {
+  SPRITE: 'Sprite',
```

Comment:
nit: Would it be more consistent to change this to PNG? That way it's talking about the type of the upload for both cases?

Reply:
Ah, good point. I was aligning these with the functions (uploadSpriteToAnimationLibrary and uploadJSONtoAnimationLibrary), if I change this to `png` and `json`, does it make sense to update the function names too? Or does it make sense to go in the opposite direction and use 'Sprite' and 'Metadata' across the board?

Reply:
I think either sprite/metadata or png/json would be fine as long as it's consistent throughout

Reply:
Sounds good! I went with sprite/metadata because I think it is more descriptive.